modules/setting/mailer.go

```diff
@@ -151,6 +152,22 @@ func loadMailerFrom(rootCfg ConfigProvider) {
 		log.Fatal("Unable to map [mailer] section on to MailService. Error: %v", err)
 	}
 
+	overrideHeader := rootCfg.Section("mailer.override_header").Keys()
+	MailService.OverrideHeader = make(map[string][]string)
+	for _, key := range overrideHeader {
+		keyName := key.Name()
+		values := key.Strings(",")
+		if keyName != "" {
+			valuesToSet := make([]string, 0, len(values))
+			for _, v := range values {
+				if v := strings.TrimSpace(v); v != "" {
+					valuesToSet = append(valuesToSet, v)
+				}
+			}
```

Comment:
What I don't quite get: Why are we parsing/splitting the header value in first place? Couldn't we just pass it as-is or am I understanding something not about mail headers?

Reply:
I just want to fix things like `value1,value2,` into `value1, value2`

Reply:
Hmm ok, but I don't think it's really good to have such magic, would just pass the user-entered value as is. Maybe they have a reason they put a trailing comma, etc.

Reply:
ok ...